genconfig/cmd/gen/main.go

```diff
@@ -394,6 +401,8 @@ func (s *katzenpost) genValidatorConfig(index int) error {
 		nodeDirName := fmt.Sprintf("auth-%d", i)
 		nodeDir := filepath.Join(s.outputDir, nodeDirName)
 		tmConfig.SetRoot(nodeDir)
+		tmConfig.P2P.AllowDuplicateIP = true
```

Comment:
Have you thought about the implications of allowing duplicate IPs with respect to privacy? Not sure if this is a good idea.

Reply:
I agree with you, it's probably not good for privacy. genconfig was made for generating testneet configurations, maybe we can add `-secure` or `-mainnet` to disable this?

Reply:
Adding `-mainnet` is a good idea.